src/components/maincontroller.js

```diff
@@ -694,14 +678,30 @@ export function setTextTrack(index) {
         if (subtitleTrack) {
             textTracksManager.setActiveByIds([subtitleTrack.trackId]);
             var subtitleAppearance = window.subtitleAppearance;
+
+            // https://developers.google.com/cast/docs/reference/chrome/chrome.cast.media.TextTrackStyle
+            var textTrackStyle = new cast.framework.messages.TextTrackStyle();
```

Comment:
this stuff doesn't work right now because I don't think it necessarily gets called

Reply:
removed from the PR